bats/fb-install-katello.bats

```diff
@@ -56,6 +56,17 @@ setup() {
   sed -ir "s/^\s*server\s*=.*/server = $(hostname -f)/g" /etc/puppet/puppet.conf
 }
 
+@test "enable haveged" {
+  # EPEL is enabled later in other scripts, just install directly for now.
+  if tIsRHEL 6; then
+    rpm -ivh "http://dl.fedoraproject.org/pub/epel/6/x86_64/haveged-1.9.1-2.el6.x86_64.rpm"
+    service haveged start
+  elif tIsRHEL 7; then
+    rpm -ivh "http://dl.fedoraproject.org/pub/epel/7/x86_64/h/haveged-1.9.1-1.el7.x86_64.rpm"
```

Comment:
I had some concern that if I installed the epel package this early in setup, it may confuse `setup.rb` later on. I did not test that doing this breaks things though.